etk-asm/src/parse/error.rs

```diff
@@ -60,6 +62,31 @@ pub enum ParseError {
         /// The location of the error.
         backtrace: Backtrace,
     },
+
+    /// An argument provided to a macro was of the wrong type.
+    #[snafu(display("File {} does not exist", path.to_string_lossy()))]
+    #[non_exhaustive]
+    FileNotFound {
```

Comment:
What returns this variant?

Reply:
I don't understand this comment, what do you mean?

Reply:
If I search through your branch, I only find this definition. I don't see the variant `FileNotFound` being used anywhere. It's entirely possible I'm just missing where it's used, but I wanted to double check before merging.